### DISCUSSION TITLE
Archive location and backup

### DISCUSSION THREAD

**Discussion by ecodiv** [Mattk70/Chirpity-Electron#176] [Q&A] [answered]
Awesome tool! One question though, one can store results in an archive, but where is the archive located? I would like to be able to backup the archive.

**Comment by Mattk70** [comment, marked as answer]
Hi @ecodiv, the database location depends on the version you have installed:

| OS       | Database Location    |
|----------|-------------|
| Windows  | %APPDATA%/chirpity/archive*.sqlite   |
| Windows  (portable) | ./chirpity-data/archive*.sqlite   |
| macOS    |  /Users/_your-username_/Library/Application Support/chirpity/archive*.sqlite |
| Linux    | ~/.config/chirpity/archive*.sqlite    |

**Reply by ecodiv** [reply]
Thanks!
